test/metabase/pulse/render/body_test.clj

```diff
@@ -410,25 +410,30 @@
 (deftest render-bar-graph-test
   (testing "Render a bar graph with non-nil values for the x and y axis"
     (is (has-inline-image?
-         (render-bar-graph {:cols default-columns
-                            :rows [[10.0 1] [5.0 10] [2.50 20] [1.25 30]]}))))
+         (render-bar-graph {:cols         default-columns
+                            :rows         [[10.0 1] [5.0 10] [2.50 20] [1.25 30]]
+                            :viz-settings {}}))))
```

Comment:
Now that I always pass `:viz-settings` to FE and it expects this value to always be present, I need to fix all the tests here.

I could make FE check if this value is null, but in reality, it will always have a value, so making that change in favor of not having to fix tests, seems a bit misleading.